### ISSUE TITLE
Hellow from G.M.B. mod on SC HD

### ISSUE THREAD

**Issue by VALROY17** [Sh0wdown/UnofficialCrusaderPatch#402]
ENG:
Hello friends! I LORD_VALROY, the author of a large mod for Stronghold Crusader. I am glad that there are still people who are ready to make content for their favorite game. I hope that you will achieve high results.
I am always glad to help you and exchange experience, since I have been engaged in the modding of this game since 2010. My result is G.M.B. mod version 4.4.0, which improves the game.
I will not write here what is in mod so as not to increase the size of the post. Detailed information is on the website moddb.com
Hope we can help each other!

RUS:
Здравствуйте друзья! Я LORD_VALROY, автор большого мода для Stronghold Crusader. Я рад, что все еще есть люди, которые готовы делать контент для своей любимой игры. Я надеюсь, что вы добьетесь высоких результатов.
Я всегда рад помочь вам и обменяться опытом, так как я занимаюсь моддингом этой игры с 2010 года. Мой результат - G.M.B. мод версии 4.4.0, который улучшает игру.
Я не буду здесь писать, что есть в моде, чтобы не увеличивать размер поста. Подробная информация на сайте [moddb.com] (https://www.moddb.com/mods/lord-valroy/news/gmb-mod-by-lord-valroy-v440-release)
Надеюсь, мы сможем помочь друг другу!

**Comment by Russianstory** [comment]
@VALROY17 ну сука наконец-то вы родили))))))))))))азазаз))) я рад тебя видел зараза ты)
Im glad to see u maaan.
Valroy has good skill in moddin Stronghold   rly  gyus!!!

**Comment by Russianstory** [comment]
Я его знаю уже давно))) он правда хорошо разбирается в модинге и в структуре Стронгхолда. Как вы, так и он будете друг другу полезны. В моём ЛИЧНО понимании мы тут все собрались чтоб в конце концов сделать супер глобальный глубокий мод на Стронгхолд Крестоносец 1 используя все разработки, наработки и возможности игры. Мод этот впитает всё самое лучше из других модов + то, что ребята делают здесь. У меня такая мечта - играть в фантастически феноменальный по крутости МОД на Стронгхолд. Да пребудет с нами - ВСЁ !)))))))

=

I know him for a long time))) he really knows a lot about modding and Stronghold’s structure. Both you and he will be helpful to each other. In my PERSONAL understanding, we all gathered here in the end to make a super global deep mod on Stronghold Crusader 1 using all the developments, developments and possibilities of the game. This mod will absorb all the best from other mods + what the guys are doing here. I have such a dream - to play fantastically phenomenal on cool MOD on Stronghold. Be with us - EVERYTHING!)))))))

**Comment by Lolasik011** [comment]
@VALROY17 О Валрой!

**Comment by Russianstory** [comment]
@Lolasik011 ИСТИНУ ГОВОРИТ. С меня как только выпущу мод забирайте все звуки из моей сборки. Я силён звуками ) поэтому их и отдаю). Так же анимация котелка возле замка там я попросил давно человека сделать анимацию бурления похлёбки ))) если приглядеться) Да можно сделать великий мод.

**Comment by GRhin** [comment]
@VALROY17 out of curiosity, do you mod the same way Sh0wdown does? or is there another way? 
Id really like if all mod creators could put down what they know somewhere in a wiki type thing to help the less adventurous of us to mod off what you have figured out already. (make menu items like this, change unit stats like that etc)

**Comment by VALROY17** [comment]
ENG:
@GRhin our approaches differ. Due to the fact that changing the executable file is prohibited (violation of the rules and copyright) I went the way of creating a separate program (launcher), which helps me to go through the restrictions in creating mods for this game.
Rus:
наши подходы отличаются. В связи с тем, что изменение исполняемого файла запрещено (нарушение правил и авторских прав), я пошел по пути создания отдельной программы (лаунчера), которая помогает мне преодолеть ограничения при создании модов для этой игры.

**Comment by Russianstory** [comment]
I will say it again. It is necessary for all to unite and collect their achievements from different mods into one big, really global mod. And add there what the host of this forum does. Each of us has its strengths and advantages.

=

Я ещё раз скажу. Надо всем объединиться и собрать свои наработки с разных модов в один большой, реально глобальный мод. И добавить туда тот, что делает хозяин этого форума. У каждого из нас есть свои сильные стороны и плюсы.

**Comment by Lolasik011** [comment]
> Я ещё раз скажу. Надо всем объединиться и собрать свои наработки с разных модов в один большой, реально глобальный мод. И добавить туда тот, что делает хозяин этого форума. У каждого из нас есть свои сильные стороны и плюсы.

Согласен! Надо собрать все фанатские наработки и запихнуть а Лаунчер. А потом каждый выберет то что ему нужно, и тем самим создаст свою сборку... Ну либо создать действительно Глобальный Проект.

**Comment by Lolasik011** [comment]
@Russianstory Я написал сегодня @Sh0wdown... Буду ждать от него ответа ...

**Comment by GRhin** [comment]
I gotta disagree with putting it in one big mod. Firstly, each has different ways of solving problems, each with pros and cons, 
Secondly, too many options in the installer actually detracts from its usability in my opinion. If I want to play with a friend, we have to go through ensuring the same settings are selected on each pc. I do prefer Valroy's menu for this, as often its a decision between groups of settings, (UCP does it for castles, an option between every's castles, or vanilla, or what have you, Others have asked to break this down into a per-AI option though which is what i am saying reduces usability for me). 
However I do agree that the knowledge should be more widely available for other modders, for the same reason that each of us has our own strengths and weaknesses, so putting new ideas into new mods would be good.
For Example, @Russianstory and I have some slightly differing ideas on how the unit balance should be updated, Id love to have enough information somewhere that I could enact my own balance update, and give to my friends so we can play the same settings, while russianstory may want to enact his balance and play that with his friends, this could be a "group of settings" option in another mod, but every time someone does their own work, whether it be in new AIs or castles, or balances, or something else, Im not sure its practical to add to the same mod every time.
Maybe what would be better would be a site like http://stronghold.heavengames.com/downloads/index.php with better mod categories and maybe an installer for the downloadable mods/patches to plug into where players can upload their options/mods/patches.

@VALROY17 is there somewhere I can go to learn how you made your mod? what would i need to learn to do so? do you have documentation of how to do certain things (like changing unit stats)

**Comment by Russianstory** [comment]
That is why it is necessary to make one big mod BUT to take into the mod only what is truly universal and fits all modifications. Sounds, bots, in general, the size of buildings, a set of buildings, soils, the number of troops, the level of AI, a set of maps, and a lot more. It is necessary to make one large universal available for all modes on top of the Crusader. And for those who want to play different mods (small ones) there is an opportunity to download them from the Internet. Politics - divide and conquer - is not always good. I am for the union.

ps I wrote you there - let's change the parameters of the longbow and we can test, and the truth may not be necessary to change the range because of the speed of firing the longbow its range will be reduced to zero as its advantage ... but received no answer)).

Lolasik - thanks I also wrote to him in the mail))) but he is silent. I will write about those mods to you in my topic =)

=

Вот поэтому надо сделать один большой мод НО взять в мод только то, что действительно универсально и подходит под все модификации. Звуки, Боты, в целом размеры зданий, набор зданий, грунты, количество войск,  уровень ИИ, набор карт, да много чего ещё. Именно надо сделать один большой универсальный доступный для всех мод поверх Крестоносца. А для тех кто хочет играть в разные моды (мелкие) есть возможность их скачать из интернета. Политика - разделяй и властвуй - это не всегда хорошо. я за объединение.

п.с. я же тебя там писал - давай изменим параметры лонгбоу и протестим может и правда не надо менять дальность ибо из-за скорости стрельбы лонгбоу дальность его будет сведена к нулю как его преимущество...но ответа не получил)).

Лоласик - спасибо я тоже ему написал на почту))) но он молчит. Про те моды отпишу тебе в своей теме =)

**Comment by Lolasik011** [comment]
> Лоласик - спасибо я тоже ему написал на почту))) но он молчит. Про те моды отпишу тебе в своей теме =)

ok
